containers/src/Kokkos_ScatterView.hpp

```diff
@@ -1248,11 +1248,11 @@ class ScatterView<DataType, Kokkos::LayoutLeft, DeviceType, Op,
                        KOKKOS_IMPL_CTOR_DEFAULT_ARG};
     arg_N[internal_view_type::rank - 1] = unique_token.size();
     internal_view                       = internal_view_type(
-        view_alloc(WithoutInitializing,
-                   std::string("duplicated_") + original_view.label(),
-                   exec_space),
-        arg_N[0], arg_N[1], arg_N[2], arg_N[3], arg_N[4], arg_N[5], arg_N[6],
-        arg_N[7]);
+                              view_alloc(WithoutInitializing,
+                                         std::string("duplicated_") + original_view.label(),
+                                         exec_space),
+                              arg_N[0], arg_N[1], arg_N[2], arg_N[3], arg_N[4], arg_N[5], arg_N[6],
+                              arg_N[7]);
```

Comment:
so weird ....

Reply:
I used the wrong version of clang-format. Will fix it.